contracts/interfaces/IAPM.sol

```diff
@@ -32,5 +32,5 @@ interface IAPM {
 
     function getAmountsOut(uint amountIn, address[] memory path) external view returns (uint[] memory amounts);
 
-    function removeLiquidity(address _to, address tokenAddress, uint amount) external;
+    function removeLiquidityBank(address _to, address tokenAddress, uint amount) external;
```

Comment:
removeLiquidityGovernance was not present in the interface so couldn't be known by Governance... but anyway u can just leave removeLiquidity and edit access in implementation